<launch>
<arg name="remapped_tf" default="/tf" />
<arg name="output" default="log" />
<param name="robot_description" textfile="$(find drchubo_v3)/robots/drchubo_v3.urdf"/>


    <include file="$(find ud_sensor_head)/launch/ud_sim_full_robot.launch" >
        <arg name="rviz_file" value="$(find hose_localization)/graspable_cylinder.rviz"/>
    </include>


    <node name="pcd_transfrom_publisher" pkg="tf" type="static_transform_publisher" args="-.7 0 0 0 0 0 /rightFoot /pcd_transform 100"  output="$(arg output)" >
      <remap from="/tf" to="$(arg remapped_tf)" />
    </node>

    <node name="pcd_publisher" pkg="pcl_ros" type="pcd_to_pointcloud" args="/home/jweisz/hose_oct8_B.pcd 10"  output="$(arg output)" >
      <remap from="/tf" to="$(arg remapped_tf)" />
      <param name="frame_id" value="/pcd_transform"/>
      <remap from="cloud_pcd" to="/rgbd_longrange/depth/points_xyzrgb"/>
    </node>

    <include file="$(find hose_localization)/launch/LaunchHoseMarkers.launch" >
    	     <arg name="output" value="screen" />
    </include>
   <node name="octomap_server" type="octomap_server_node" pkg="octomap_server">
   	 <remap from="cloud_in" to="/rgbd_longrange/depth/points_xyzrgb"/>
	 <param name="frame_id" value = "/leftFoot"/>
	 <param name="base_frame_id" value = "/leftFoot"/>
    </node>		     
	 

</launch>
